pixi.toml

```diff
@@ -0,0 +1,48 @@
+[workspace]
+authors = ["ManifoldFR"]
+channels = ["conda-forge"]
+name = "nanoeigenpy"
+description = "A support library for bindings between Eigen/C++ and Python, based on nanobind"
+platforms = ["linux-64", "osx-arm64", "osx-64"]
+version = "0.1.0"
+
+[tasks]
+configure = { cmd = [
+  "CXXFLAGS=$NANOEIGENPY_CXX_FLAGS",
+  "cmake",
+  "-G",
+  "Ninja",
+  "-B",
+  "build",
+  "-S",
+  ".",
+  "-DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX",
+  "-DCMAKE_BUILD_TYPE=$NANOEIGENPY_BUILD_TYPE",
+] }
+build = { cmd = "cmake --build build --target all", depends-on = ["configure"] }
+clean = { cmd = "rm -rf build" }
+
+[dependencies]
+python = ">=3.9"
+eigen = ">=3.3.1"
+
+[feature.python-latest.dependencies]
+python = "3.13.*"
+
+[feature.python-oldest.dependencies]
+python = "3.9.*"
+
+
+[build-dependencies]
+nanobind = ">=2.5.0,<3"
+cmake = ">=3.15"
+cxx-compiler = ">=1.7"
+ccache = ">=4.11.2"
+ninja = ">=1.12.1"
+pkg-config = ">=0.29.2"
+git = ">=2.49.0"
```

Comment:
Are you sure you need git ?
Usually, it can be required by fetch content on stubgen.

Reply:
We'll need it for when users `FetchContent` the cmake modules, no? (although we don't support it yet)